src/main/resources/messages.properties

```diff
@@ -19,6 +19,8 @@ general.check-all-that-apply=Check all that apply.
 general.you=(You)
 general.none-of-the-above=None of the above
 general.good-news=Good news!
+general.i-dont-know=I don't know
+general.not-interested=No, I\u2019m not interested
```

Comment:
Just curious why line 22 uses an apostrophe and line 23 uses unicode

Reply:
Sometimes Figma c/p has a unicode apostrophe 🤷